### ISSUE TITLE
question: plug simulation

### ISSUE THREAD

**Issue by foxthefox** [peuter/ecoflow#5]
I have a question to the implementation of the plug handling. I saw "simulation" in the code. Does it mean that a virtual device is implemented and via this the dynamic watts in the powerstream can be additionally set?

**Comment by peuter** [comment]
That was the idea, but unfortunately I did not find a way to make it work.

**Comment by foxthefox** [comment]
It would be so cool to inject the dynamic watts. Most likely it happens on BLE communication :-(

**Comment by foxthefox** [comment]
I have made some thoughts and have to ask again. Did you try to similate the smart plug with the  identical id# of a previously living plug which is removed from socket, but immediately the simulation keeps the heartbeat running?

**Comment by peuter** [comment]
Yes I tried with an ID from an existing smart plug that wasn't plugged in when I started the test. But I do not remember the amount of time that has passed between unplugging the real device and starting the test. Maybe a couple of minutes.

**Comment by foxthefox** [comment]
Thanks for clarification

**Comment by foxthefox** [comment]
now I have also a smart plug and played with it, there was an firmware update and it said that the bluetooth is switched off after connection to wlan (for better stability of wlan). it seems really so, the ble-scan showed no longer the plug.

so I assume that the update of dynamicWatts must be handled via an unknown mqtt-topic.
it would be good to find that topic somehow
I guess then the simulation would work

at least I have observed one bit in the power stream toggling with plug availability
Edit: InverterHeartbeat2 -> X_unknown_44
